demo-client/src/main/java/com/illia/client/service/FileHandlingProxyService.java

```diff
@@ -0,0 +1,58 @@
+package com.illia.client.service;
+
+import com.illia.client.config.ClientConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+@Service
+public class FileHandlingProxyService {
```

Comment:
the other one should be a proxy(as it would be a library proxy), this is regular one